test/rm.js

```diff
@@ -57,16 +60,23 @@ test('invalid option', t => {
   t.is(result.stderr, 'rm: option not recognized: @');
 });
 
-test('remove symbolic link to a dir without -r fails', t => {
-  utils.skipOnWin(t, () => {
-    const result = shell.rm(`${t.context.tmp}/rm/link_to_a_dir/`);
-    t.truthy(shell.error());
-    t.is(result.code, 1);
-    t.is(result.stderr, 'rm: path is a directory');
-    t.truthy(fs.existsSync(`${t.context.tmp}/rm/link_to_a_dir`));
-    t.truthy(fs.existsSync(`${t.context.tmp}/rm/a_dir`));
-  });
-});
+test(
+  'cannot remove a symbolic link to a directory with trailing slash without -r flag',
+  t => {
+    utils.skipOnWin(t, () => {
+      // the trailing slash signifies that we want to delete the source
+      // directory and its contents, which can only be done with the -r flag
```

Comment:
> to delete the source directory and its contents

Can you rephrase this as "delete the contents of the source directory, without removing the source directory itself or the link"?